### ISSUE TITLE
Header/Footer Logos for High Contrast Mode

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17153]
## Status
**[2024-03-11]** Moved to Icebox while OFCIO conducts expanded research; this ticket may end up obsolete depending on decisions made after further research and decisions are made. 

## Description
In the new header/footer, there are 4 logos being used. 3 of these have issues in various high contrast themes:

- Header USA logo - in Windows and iOS mobile themes you can only see "US", in Mac you are able to see the entire logo still
- Footer USA logo - In iOS mobile high contrast mode you can't see it at all, in Windows and Mac you can
- Footer Seal -  In iOS mobile high contrast mode you can't see it at all, in Windows and mac you can

## Possible Solutions
One option would be that if we were to create image files with a baked-in background, these logos would be visible when HCM was used, in the instances where it currently isn't, however we would want to make sure that it was still visible in the instances without issues. (The USA logo would need to have a white background and the footer logos would need to have a black background)

Example of this being done in the wild is on [hhs.gov](https://www.hhs.gov/):
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/a2fd3b83-a58f-499a-8743-72c00ccf939d)

Another option would be to render the logos with an `<svg>` tag instead of an `<img>` tag We would be able to specify different colors for HCM but I think this might be a bit of work. 

### Screenshots
**Windows HCM**
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/35f234f0-48e7-46f4-9416-12740086e84b)

**iPhone Smart Invert & Classic Invert (Safari and Chrome)**
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/7dd5f426-334c-4df7-adbe-ccb468bf6a4b)

## Acceptance Criteria
- [ ] Determine path forward for rendering logos visible in HCM
- [ ] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [x] Public Websites
- [ ] Facilities
- [ ] Accelerated Publishing

**Comment by aklausmeier** [comment]
@laflannery While exploring a solution, we will need to retain support for screen magnifiers. The current svg files can be zoomed in without losing resolution and becoming pixelated.

**Comment by laflannery** [comment]
Info sent to OFCIO to decide if they want to provide a new logo for USA branding

**Comment by FranECross** [comment]
Per Dave Conlon, the Federal Header/Footer improvement effort has been closed by the Whitehouse / 18f. All related tickets may be closed.
